Transfer: add support for outputs import / export

selsta opened the pull request:
Requires https://github.com/monero-project/monero/pull/7678

Resolves #2816

rating89us commented on pages/Transfer.qml:
```diff
@@ -843,6 +843,32 @@ Rectangle {
+                importOutputsDialog.open();
+            }
+            tooltip: {
+                var header = qsTr("Required for cold wallets to to sign their corresponding key images") + translationManager.emptyString;
```
```suggestion
                var header = qsTr("Required for cold wallets to sign their corresponding key images") + translationManager.emptyString;
```

selsta replied:
oops, thanks :)

selsta commented:
@rating89us please review / approve

rating89us commented:
I'm getting this error:
`W qrc:/pages/Transfer.qml:1107: TypeError: Type error`

selsta commented:
@rating89us how did you get this message?

rating89us commented:
On view-only wallet, I clicked the export button. A file browser window opened and I entered a filename, when I click on the save button, no file was created and this error message appears on console.
I'm using Ubuntu.

rating89us commented:
I didn't import https://github.com/monero-project/monero/pull/7678 before testing it. Is it included in master?

selsta commented:
> I didn't import monero-project/monero#7678 before testing this PR. Is it included in master?

It's included in v0.17.2.3 and master. Will test in view only wallet.

selsta commented:
@rating89us Thanks, found an issue. Was able to export fine now.

rating89us commented:
1) Output exporting is working now on view-only wallet. I suggest we add a feedback for the user:
`appWindow.showStatusMessage(qsTr("Outputs successfully exported to file") + translationManager.emptyString, 3);`

2) When importing the outputs file in cold wallet, I get this error:
`W qrc:/pages/Transfer.qml:1121: TypeError: Property 'importOutputsDialog' of object Wallet(0x7f342a9d7b20) is not a function`

selsta commented:
> Output exporting is working now on view-only wallet. I suggest we add a feedback for the user:

Fixed

> When importing the outputs file in cold wallet, I get this error:

Fixed.. had copy paste error

selsta commented:
Updated on more time to print success / error message.

rating89us commented:
Now it's working for me.

I think we should also print success / error message for importing/exporting key images.

selsta commented:
Also updated error string here.
